lib/index.js

```diff
@@ -27,16 +27,14 @@ function nodeRequire(module) {
 function getContext() {
     if (typeof process === 'object' && process + '' === '[object process]') {
         return process.type === 'renderer' ? 'electron' : 'nodejs';
-    }        
+    }
 
-    if (typeof navigator !== 'undefined') {
-        if (navigator.product === 'ReactNative') { // eslint-disable-line no-undef
-            return 'reactnative';
-        }
+    if (typeof jest === 'object') {
+        return 'nodejs';
+    }
 
-        if (/Chrome/.test(navigator.userAgent)) { // eslint-disable-line no-undef
-            return 'chromedebugger';
-        }
+    if (typeof Realm !== 'undefined') {
```

Comment:
It matters because during chrome debugging, `navigator.product` is now also set to `ReactNative` so if we just test for that first, `getContext()` will report `reactnative` also in chrome. I'll add a comment.